test/integration/oci_registry_source_test.go

```diff
@@ -33,9 +33,6 @@ func TestOciRegistrySourceMetadata(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unable to get image: %+v", err)
 	}
-	if err := img.Read(); err != nil {
```

Comment:
Ultimately, `stereoscope.GetImage` above will call `img.Read` before returning and also return any error from the call